tests/tdastro/utils/make_fake_opsim.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+
+import numpy as np
+from tdastro.astro_utils.opsim import OpSim
+
+
+def make_sampled_opsim(ra_vals, dec_vals, time_step, num_visits):
+    """Create a fake OpSim data that scans a grid of (ra, dec) multiple times.
+
+    Note
+    ----
+    This is a toy OpSim with lots of simplifying assumptions. It should not be
+    used for scientific validation.
+
+    Parameters
+    ----------
+    ra_vals : `numpy.ndarray`
+       The values of RA to sample (in degrees).
+    dec_vals : `numpy.ndarray`
+       The values of dec to sample (in degrees).
+    time_step : `float`
+        The time between observations (in days).
+    num_visits : `int`
+        The number of times the full grid is sampled.
+    """
+    opsim_data = {
+        "observationStartMJD": [],
+        "fieldRA": [],
+        "fieldDec": [],
+    }
+
+    for day in range(num_visits):
+        t = 60676.0 + 5.0 * day  # Start date = Jan 1, 2025
+        for ra in ra_vals:
+            for dec in dec_vals:
+                opsim_data["observationStartMJD"].append(t)
+                opsim_data["fieldRA"].append(ra)
+                opsim_data["fieldDec"].append(dec)
+                t += time_step
+
+    # Add other fields
+    num_obs = len(opsim_data["observationStartMJD"])
+    opsim_data["zp_nJy"] = np.ones(num_obs)
+    opsim_data["filter"] = np.full((num_obs), "r")
```

Comment:
Can we add another filter? 
```suggestion
    opsim_data["filter"] = np.full((num_obs), "r")
    opsim_data["filter"][0::2] = "g"
```